nx-dev/ui-customers/src/lib/customer-testimonial-carousel.tsx

```diff
@@ -0,0 +1,400 @@
+      },
+      {
+        value: 'Seamless adoption',
+        label:
+          'Nx Cloud integration delivers continuous benefits from day one.',
+      },
+      {
+        value: 'Expert guidance',
+        label:
+          'Nx Enterprise support provides quick, informed assistance on demand.',
+      },
+    ],
+    company: 'SiriusXM',
+    videoId: 'Q0ky-8oJcro',
+    thumbnail: '/images/customers/video-story-siriusxm.avif',
+    logo: {
+      icon: SiriusxmIcon,
+      height: 'h-32',
+      width: 'w-32',
+    },
+  },
+  {
+    title: 'Customer story',
+    subtitle:
+      'From 5 days to 2 hours: How Payfit improved velocity and offloads complexity.',
+    metrics: [
+      {
+        value: 'From 5 days → 2 hours',
+        label: 'Nx & Nx Cloud drastically accelerate feature deployment',
+      },
+      {
+        value: 'Eliminated CI Complexity',
+        label: 'Nx Cloud offloads CI load balancing headaches.',
+      },
+      {
+        value: 'Faster time-to-market',
+        label:
+          'Nx boosts velocity, helping teams deliver faster and more reliably.',
+      },
+    ],
+    company: 'Payfit',
+    videoId: 'Vdk-tza4PCs',
+    thumbnail: '/images/customers/video-story-payfit.avif',
+    logo: {
+      icon: PayfitIcon,
+      height: 'h-16',
+      width: 'w-16',
+    },
+  },
+  {
+    title: 'Customer story',
+    subtitle:
+      'How UKG reduced build times while scaling development across teams.',
+    metrics: [
+      {
+        value: 'From 1 day → instant builds',
+        label: 'Nx Cloud slashed build wait times, enabling dev productivity.',
+      },
+      {
+        value: 'Eliminated CI Maintenance',
+        label:
+          'Nx Cloud frees teams from managing CI, letting devs focus on code.',
+      },
+      {
+        value: 'Reduced duplication',
+        label:
+          'Shared libraries cut down redundant code across mobile and web apps.',
+      },
+    ],
+    company: 'UKG',
+    videoId: 'rSC8wihnfP4',
+    thumbnail: '/images/customers/video-story-ukg.avif',
+    logo: {
+      icon: UkgIcon,
+      height: 'h-20',
+      width: 'w-20',
+    },
+  },
+  {
+    title: 'Customer story',
+    subtitle: 'How Broadcom stays efficient and nimble with monorepos',
+    metrics: [
+      {
+        value: '2x faster',
+        label: 'Nx doubles speed, supporting fast, nimble development.',
+      },
+      {
+        value: 'Unmatched DX',
+        label: 'Nx empowers teams to be more productive than ever.',
+      },
+      {
+        value: 'Always ahead',
+        label:
+          'Nx’s rapid feature delivery keeps teams at the cutting edge of development.',
+      },
+    ],
+    company: 'Broadcom (VMware)',
+    videoId: 'RWTgYNKqxNc',
+    thumbnail: '/images/customers/video-story-broadcom.avif',
+    logo: {
+      icon: VmwareIcon,
+      height: 'h-28',
+      width: 'w-28',
+    },
+  },
+];
+
+export function CustomerTestimonialCarousel(): JSX.Element {
```

Comment:
Can you add the swipe behaviour for mobile?

Reply:
Mentioned this to Juri about adding this later if possible reusing the carousel on the enterprise page.